packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/README.md

```diff
@@ -472,6 +472,19 @@ The `lib_auth_matrix` brick contains classes, repositories, datasources and widg
 The project comes preconfigured with [Fastlane][fastlane_lnk] which allows building and deploying of android and iOS apps. All the necessary code can be found inside the `{app_directory}/fastlane/Fastfile` file. You may need to configure additional project related settings before it can run successfully (such as certificates, credentials, provisioning profiles, team id,...).
 
 For more information on how to configure your Fastfile, please check out [this example][booking_app_lnk].{{/cicd}}
+{{#cicd_github}
```

Comment:
Please fix the compiling issue. I guess the you have missed one closing bracket -> {{#cicd_github}}